frame/dvm/Cargo.toml

```diff
@@ -39,10 +39,12 @@ sp-std           = { default-features = false, git = "https://github.com/darwini
 [dev-dependencies]
 # crates.io
 array-bytes = { version = "1.3.3" }
+ethabi      = { git = "https://github.com/darwinia-network/ethabi.git", branch = "xavier-no-std", default-features = false }
+hex         = "0.4.3"
```

Comment:
why hex?

Reply:
`hex` is used in file `tests.rs` -> `hex::encode(wkton_balance_input(sender.address)))`

Reply:
why not `array_bytes::bytes2hex`

Reply:
It probably works, let me have a try.